client/src/components/myBookings/myBookings.jsx

```diff
@@ -17,8 +17,9 @@ class MyBookings extends React.Component {
     console.log('CLICKED DELETED')
   }
 
-  componentDidMount() {
-    fetch(`/my-bookings/${this.props.userId}`)
+  componentDidMount () {
+    console.log('PPROPS.USEID', this.props.user_id)
```

Comment:
typo? PPROPS.USEID

Reply:
@jenniferjinyoungyang hahaha...i am sorry.... i was in a hurry...will remove it..